### PR TITLE
[connman-qt] add new technology signals.

### PR DESCRIPTION
**Pull request by lpotter**
this needs a patched connman and wpa_supplicant.

### CODE REVIEW

File: libconnman-qt/networktechnology.cpp [hunk 1]
```diff
@@ -162,7 +172,12 @@ void NetworkTechnology::emitPropertyChange(const QString &name, const QVariant &
       Q_EMIT tetheringIdChanged(value.toString());
     } else if (name == TetheringPassphrase) {
       Q_EMIT tetheringPassphraseChanged(value.toString());
-    }
+    } /*else if (name == DhcpConnected) {
```
Comment (thp) [inline libconnman-qt/networktechnology.cpp]: Either not add it or uncomment it?

**Comment by thp** [comment]
Added one minor nitpick about the comments. Other than that, LGTM. Feel free to merge if this is still needed.

**Comment by thp** [comment]
Merge or drop?

**Comment by lpotter** [comment]
closed because we are not using those special connman signals/patch

**Pull request closed without merging**